elixir/apps/web/lib/web/live/groups/index.ex

```diff
@@ -26,80 +26,81 @@ defmodule Web.Groups.Index do
       <.breadcrumb path={~p"/#{@account}/groups"}><%= @page_title %></.breadcrumb>
     </.breadcrumbs>
 
-    <.header>
+    <.section>
       <:title>
-        All groups
+        Groups
       </:title>
-      <:actions>
+      <:action>
         <.add_button navigate={~p"/#{@account}/groups/new"}>
-          Add a new group
+          Add Group
         </.add_button>
-      </:actions>
-    </.header>
-    <!-- Groups Table -->
-    <div class="bg-white dark:bg-gray-800 overflow-hidden">
-      <div :if={Enum.empty?(@groups)} class="text-center align-middle pb-8 pt-4">
-        <h3 class="mt-2 text-lg font-semibold text-gray-900">There are no groups to display.</h3>
+      </:action>
+      <:content>
+        <div class="bg-white dark:bg-gray-800 overflow-hidden">
+          <div :if={Enum.empty?(@groups)} class="text-center align-middle pb-8 pt-4">
```

Comment:
@bmanifold I think in places like this one we should use `:empty` slot that you added to table components so that we don't have multiple ways to render empty tables.

Reply:
Sounds good.  I knew this PR was going to have a lot of lines of diff, so didn't want to add more to it, but I'm happy to update the existing tables.

Reply:
I went back and updated all the the `<.table>` components that were used to have an `<:empty>` slot.